packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -44,6 +44,11 @@ export default class QuanticFacet extends LightningElement {
   /** @type {boolean} */
   @api noSearch = false;
   /** @type {string} */
+  @api delimitingCharacter = '<';
```

Comment:
As far as I know, the default delimiting character in the index is still `;`, so this default value will not work out of the box for most multi-value fields.

Reply:
Atomic and headless set '<' as the default.

Reply:
Haha actually it's `>` 😅  so i'll change it but not to `;`

Reply:
I might be missing something, but it surely does not match the [platform documentation regarding multi-value fields](https://docs.coveo.com/en/2036/index-content/about-fields#multi-value-fields).

Reply:
@olamothe do you know what's up with this?

Reply:
From what I understand, the `delimitingCharacter` default value could have been confused between the facet and the category facet.

`;` have always been the default delimiter for multi-value fields. IMO it should be the default for the facet. The category facet though has a second delimiting character to split the value path. *Maybe* this one should default to `>`, but I'm still not sure as the [JS UI documentation is using pipes as a delimiter](https://docs.coveo.com/en/2667/javascript-search-framework/use-hierarchical-facets).

So, I really don't understand how we ended up with `>` as the default delimiting character for the facet.

Reply:
Now I understand what's going on. I got confused with the `multiValueFacetTokenizers` field setting that defines the multi-value delimiter for a field. This value is not configurable from the Admin UI but can still be configured through the API. But, this delimiter appear to be handled internally by the platform (index and/or search API) since the values are returned as an array.

So, in this case the `delimitingCharacter` option refers to the "level" delimiter and not the "multi-value" delimiter like I thought.